src/renderer/screens/exchange/Swap2/Form/TargetAccountDrawer/index.js

```diff
@@ -23,10 +23,13 @@ import { rgba } from "~/renderer/styles/helpers";
 
 const AccountWrapper = styled(Tabbable)`
   cursor: pointer;
+  &:hover {
+    background-color: ${p => p.theme.colors.palette.text.shade10};
+  }
   ${p =>
     p.selected
       ? `
-    background-color: ${p.theme.colors.lightGrey};
+    background-color: ${p.theme.colors.palette.background.main};
```

Comment:
You sure about this one?, either `background.default` or `background.paper`
```
type RawPalette = {
  type: "light" | "dark",
  primary: {
    main: string,
    contrastText: string,
  },
  secondary: {
    main: string,
  },
  divider: string,
  background: {
    paper: string,
    default: string,
    wave: string,
  },
  action: {
    active: string,
    hover: string,
    disabled: string,
  },
  wave: string,
};
```

Reply:
Damn, I can't understand why I used `main` and why the type system didn't alert me.
Fixed, thanks 👍